lib/bin_conf/Bin_conf.ml

```diff
@@ -535,11 +535,13 @@ let global_lib_term =
         new_global.lib_conf )
     $ global_term )
 
-let update_using_cmdline info config =
-  match
-    Cmd.eval_value ~err:discard_formatter ~help:discard_formatter
-      (Cmd.v info global_lib_term)
-  with
+let global_lib_term_eval = lazy (
```

Comment:
You need to format the code with `dune fmt`. Do you also want to add yourself to `CHANGES.md` ?